src/backend/persons/main.go

```diff
@@ -70,7 +70,7 @@ func getPersons(table dynamo.Table) events.APIGatewayProxyResponse {
 }
 
 func addPerson(table dynamo.Table, reqBody string) events.APIGatewayProxyResponse {
-	id, _ := uuid.NewUUID()
```

Comment:
https://godoc.org/github.com/google/uuid#NewUUID

> NewUUID returns a Version 1 UUID based on the current NodeID and clock sequence, and the current time. If the NodeID has not been set by SetNodeID or SetNodeInterface then it will be set automatically. If the NodeID cannot be set NewUUID returns nil. If clock sequence has not been set by SetClockSequence then it will be set automatically. If GetTime fails to return the current NewUUID returns nil and an error.
> 
> **In most cases, New should be used.**

https://godoc.org/github.com/google/uuid#New

```
func New() UUID
```